src/xdocs/config_filters.xml

```diff
@@ -2232,6 +2232,275 @@ public class UserService {
+        <div class="wrapper">
+          <table>
+            <tr>
+              <th>name</th>
+              <th>description</th>
+              <th>type</th>
+              <th>default value</th>
+              <th>since</th>
+            </tr>
+            <tr>
+              <td>nearbyTextPattern</td>
+              <td>Specify nearby text pattern to trigger filter to begin suppression.</td>
+              <td><a href="property_types.html#Pattern">Pattern</a></td>
+              <td><code>"SUPPRESS CHECKSTYLE (\w+)"</code></td>
+              <td>10.10.0</td>
+            </tr>
+            <tr>
+              <td>checkPattern</td>
+              <td>Specify check name pattern to suppress. Property can
+                also be a RegExp group index at <code>nearbyTextPattern</code>
+                in format of <code>$x</code> and be picked from line
+                that matches <code>nearbyTextPattern</code>.</td>
+              <td><a href="property_types.html#Pattern">Pattern</a></td>
+              <td><code>".*"</code></td>
+              <td>10.10.0</td>
+            </tr>
+            <tr>
+              <td>messagePattern</td>
+              <td>Specify check violation message pattern to suppress.</td>
+              <td><a href="property_types.html#Pattern">Pattern</a></td>
+              <td><code>null</code></td>
+              <td>10.10.0</td>
+            </tr>
+            <tr>
+              <td>idPattern</td>
+              <td>Specify check ID pattern to suppress.</td>
+              <td><a href="property_types.html#Pattern">Pattern</a></td>
+              <td><code>null</code></td>
+              <td>10.10.0</td>
+            </tr>
+            <tr>
+              <td>lineRange</td>
+              <td>Specify negative/zero/positive value that defines the number of lines
+                preceding/at/following the suppressing nearby text. Property can
+                also be a RegExp group index at <code>nearbyTextPattern</code>
+                in format of <code>$x</code> and be picked from line
+                that matches <code>nearbyTextPattern</code>.</td>
+              <td><a href="property_types.html#String">String</a></td>
+              <td><code>"0"</code></td>
+              <td>10.10.0</td>
+            </tr>
+          </table>
+        </div>
+      </subsection>
+      <subsection name="Examples" id="SuppressWithNearbyTextFilter_Examples">
+        <p>
+          To configure the filter to suppress audit events on the same line:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;/&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;MagicNumber&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example:
+        </p>
+        <source>
+int a = 1; // SUPPRESS CHECKSTYLE the reason for this is
+int b = 1; // violation
```

Comment:
https://checkstyle.org/config_coding.html#MagicNumber_Properties are you sure it is violation?

Make values to be 45.

Update all.

Reply:
These slipped. Done now.